testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,69 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev python3-dev freeglut-dev glew-dev"
+install=""
+subpackages="$pkgname-python2:py2 $pkgname-python3:py3 $pkgname-dev $pkgname-utils"
+source="$pkgname-$pkgver.tar.gz::http://github.com/imageworks/OpenColorIO/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/OpenColorIO-$pkgver
+
+build() {
+	cd "$builddir"
+
+	for pyver in python3 python2; do
+		mkdir build-$pyver && cd build-$pyver
+
+		cmake .. \
+			-DCMAKE_INSTALL_PREFIX=/usr \
+			-DPYTHON=/usr/bin/$pyver \
+			-DOCIO_BUILD_TRUELIGHT=OFF \
+			-DOCIO_BUILD_NUKE=OFF \
+			|| return 1
+		make
+		cd ..
+	done
+}
+
+package() {
+	cd "$builddir"/build
+	make DESTDIR="$pkgdir" install
+
+	for pyver in python3 python2; do
+		cd "$builddir"/build-$pyver
+		make DESTDIR="$pkgdir" install
+	done
+}
+
+utils() {
+	pkgdesc="OpenColorIO color management framework utilities"
+	mkdir -p "$subpkgdir"/usr/
+	mv "$pkgdir"/usr/bin "$subpkgdir"/usr || return 1
+}
+
+_python() {
+	pkgdesc="Python $1 bindings for OpenColorIO color management framework"
+	depends="python$1"
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/python$2 "$subpkgdir"/usr/lib/
+}
+
+py2() {
+	PY2_VERSION=$(python -c 'import sys; print("%i.%i" % (sys.version_info.major, sys.version_info.minor))')
```

Comment:
Or you can entirely avoid it as in your second package using `mv "$pkgdir"/usr/lib/python$1?? "$subpkgdir"/usr/lib/` (or simply `$pkgdir"/usr/lib/python$1*`).